### DISCUSSION TITLE
DWIN_SET (Gotcha) with v2.0.0

### DISCUSSION THREAD

**Discussion by jetel73** [Jyers/Marlin#1518] [Q&A] [answered]
Hello do I need to return to DWIN_SET (Original) for the screen to work properly with this new firmware or Gotcha's is still ok ?

**Comment by Jyers** [comment, marked as answer]
Gotcha should still work as expected :)

**Reply by Cal-LF** [reply]
@Jyers I noticed the define custom icons is no longer in the config.h

**Reply by Jyers** [reply]
Ah yes, that must not have made it into the default marlin configs. I believe it's still implemented though. So you should be able to add that define in and it will work. Let me know if there are any issues.
